homeassistant/components/plugwise/climate.py

```diff
@@ -1,227 +1,214 @@
 """Plugwise Climate component for Home Assistant."""
 
 import logging
+from typing import Dict
 
-import haanna
-import voluptuous as vol
+from Plugwise_Smile.Smile import Smile
```

Comment:
Remove, comes in via `hass.data`

Reply:
`PlugwiseError` is used for exception-handling